docs/rules/arrow-spacing.md

```diff
@@ -22,7 +22,10 @@ The default configuration is `{ "before": true, "after": true }`.
 
 `true` means there should be **one or more spaces** and `false` means **no spaces**.
 
-The following patterns are considered problems if `{ "before": true, "after": true }`.
+### both true
```

Comment:
I think it might be better to do "before: true, after:true" to follow the same convention you have later in this file.